spec/transaction.test.js

```diff
@@ -22,8 +23,11 @@ describe("Transaction", () => {
   //use mock to test date
   describe("#date", () => {
     it("stores the date when the deposit was made", () => {
-      let transaction = new Transaction(5.16);
+      const date = "Mon Mar 30 2020 00:00:00 GMT+0100 (British Summer Time)";
+      set(date);
```

Comment:
nifty, very cool mocking library!

Reply:
Moment is the same one I used it is indeed nifty! LGTM :)

Reply:
I like this a lot!